### PR TITLE
support for digitalocean spaces by adding endpoint in s3 config

### PR DESCRIPTION
**Pull request by mubin986**

### CODE REVIEW

**Comment by sgreene570** [comment]
@mubin986 btw, this PR overlaps with https://github.com/shallwefootball/upload-s3-action/pull/42

A more generic solution most likely makes more sense, but curious to hear your thoughts!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Upload S3 ☁️
+# Upload S3 / DO Spaces ☁️
```
Comment (shallwefootball) [inline README.md:1]: No need "DO Spaces"

File: README.md [hunk 2]
```diff
@@ -79,6 +80,8 @@ The following settings must be passed as environment variables as shown in the e
 | `aws_bucket`            | (Required) The name of the bucket you're upload to.          |
 | `source_dir`            | (Required) The local directory (or file) you wish to upload to S3. The directory will replace to key generated by [shortid](https://github.com/dylang/shortid) in S3 |
 | `destination_dir`       | (Optional) The destination directory in S3<br />If this field is excluded a [shortid](https://github.com/dylang/shortid) will be generated |
+| `endpoint`       | (Optional) If you want to use Digitalocean Spaces, then you have to provide the endpoint. e.g.: sgp1.digitaloceanspaces.com |
```
Comment (shallwefootball) [inline README.md:83]: add empty space

**Comment by shallwefootball** [comment]
thx 👍 
https://github.com/shallwefootball/upload-s3-action/releases/tag/v1.3.0

**Pull request closed without merging**